core-v1/src/main/scala/dsl/WrapExprDsl.scala

```diff
@@ -0,0 +1,22 @@
+//package com.rallyhealth.vapors.v1
+//
+//package dsl
+//
+//import algebra.{ExprHL2, ExprHNil2}
+//
+//import shapeless.{::, HNil}
+//
+//trait WrapExprDsl {
+//  self: BuildExprDsl with DslTypes =>
+//
+//  def wrap[I, A, B](
+//    a: I ~:> W[A],
+//    b: I ~:> W[B],
+//  ): ExprHL2[I, W[A] :: W[B] :: HNil, OP] = a :: b :: ExprHNil2[OP]
+//
+//  def wrap[I, A, B, C](
+//    a: I ~:> W[A],
+//    b: I ~:> W[B],
+//    c: I ~:> W[C],
+//  ): ExprHL2[I, W[A] :: W[B] :: W[C] :: HNil, OP] = a :: b :: c :: ExprHNil2[OP]
+//}
```

Comment:
I don't think this syntax is necessary since I was able to get `::` working... however, it might be good to add for backwards compatibility. I should either remove this commented code or add backwards compatibility depending on whether the syntax is helpful.